WcaOnRails/app/models/registration.rb

```diff
@@ -25,21 +27,11 @@ def pending?
     end
   end
 
-  before_save :normalize_event_ids
   private def normalize_event_ids
     self.eventIds = events.map(&:id).join(" ")
   end
 
-  attr_writer :birthday
-  def birthday
-    birthYear == 0 || birthMonth == 0 || birthDay == 0 ? nil : Date.parse("%04i-%02i-%02i" % [ birthYear, birthMonth, birthDay ])
-  end
-
-  before_validation :unpack_dates
   private def unpack_dates
-    if @birthday.nil? && !birthday.blank?
```

Comment:
Removing this block of code worries me. If someone is saving a Registration without setting the `@birthday` attribute, it looks like we'll go into line 36 and clear the `birthYear`, `birthMonth`, and `birthDay` attributes. Do tests still pass with this removed?